comms/dht/tests/dht.rs

```diff
@@ -536,6 +540,85 @@ async fn dht_propagate_dedup() {
     assert_eq!(count_messages_received(&received, &[&node_C_id]), 1);
 }
 
+#[tokio_macros::test]
+#[allow(non_snake_case)]
+async fn dht_repropagate() {
+    // Node C knows no one
+    let mut node_C = make_node(PeerFeatures::COMMUNICATION_NODE, []).await;
+    // Node B knows about Node C
+    let mut node_B = make_node(PeerFeatures::COMMUNICATION_NODE, [node_C.to_peer()]).await;
+    // Node A knows about Node B and C
+    let mut node_A = make_node(PeerFeatures::COMMUNICATION_NODE, [node_B.to_peer(), node_C.to_peer()]).await;
```

Comment:
Is the intention not that (1) B sends to C, then (2) A sends the same message to B, who will propagate it to C, so that C will receive a duplicate message?

Reply:
Correct, this test is not complete but does test that the message is received more than once